lib/bittrex.rb

```diff
@@ -12,6 +12,11 @@ module Bittrex
   autoload :Wallet,        'bittrex/wallet'
   autoload :Withdrawal,    'bittrex/withdrawal'
 
+  def self.extract_timestamp value
```

Comment:
Please follow the declaration styling of the rest of the repo `def self.extract_timestamp(value)`

Reply:
done!